test/controllers/uploads_controller_test.rb

```diff
@@ -1,91 +1,110 @@
+# frozen_string_literal: true
 require 'test_helper'
+require 'mocha/mini_test'
 
 class UploadsControllerTest < ActionController::TestCase
   setup do
     unzip_if_necessary('measures')
     collection_fixtures('measures', 'bundles')
-    BUNDLES['2016'] ||= HealthDataStandards::CQM::Bundle.find_by(:version.in => ['2.7.0'])
+    bundle = HealthDataStandards::CQM::Bundle.find_by(:version.in => ['2.7.0'])
+    # bundle_method = mock
+    # bundle_method.stubs(:[]).returns(bundle)
```

Comment:
Can above two commented lines be removed?

Reply:
Good catch, will do.